app/components/companion_windows/external_link_component.rb

```diff
@@ -10,5 +10,11 @@ def initialize(viewer:, label:)
     attr_reader :viewer, :label
 
     delegate :external_url, to: :viewer
+
+    def stimulus_controllers
+      return '' unless viewer.instance_of?(::Embed::Viewer::Geo)
+
+      "data-controller=meta-check data-meta-check-url-value=#{viewer.purl_object.meta_json_url}"
```

Comment:
```suggestion
      "data-controller=\"meta-check\" data-meta-check-url-value=\"#{viewer.purl_object.meta_json_url}\""
```

Reply:
This breaks things. When you do this it adds double quotes.
![Screenshot 2025-03-06 at 2 31 20 PM](https://github.com/user-attachments/assets/69081bce-ffb0-4791-8eef-f7aa2c5aadb9)

When you do what I have it renders properly:
![Screenshot 2025-03-06 at 2 32 22 PM](https://github.com/user-attachments/assets/84ae5cdc-badb-4815-86a1-93636495dc11)

Reply:
Try adding `.html_safe` to the string.

Reply:
Why can't we leave it as is? It is rendering correctly and html_safe isn't something we really like doing unless we have to which we don't

Reply:
Yeah, I went with link_to. I think it is cleaner and clearer.